src/schedlib/policies/sat.py

```diff
@@ -615,6 +621,24 @@ def apply(self, blocks: core.BlocksTree) -> core.BlocksTree:
 
         blocks = core.seq_sort(blocks['baseline']['cmb'] + blocks['calibration'], flatten=True)
 
+        # add hwp direction to cal blocks
+        if self.hwp_override is None:
+            for i, block in enumerate(blocks):
+                if block.subtype=='cal':
```

Comment:
I feel like maybe we should check if the cal block has a set HWP direction before writing over it? Seems like that will protect us from late problems when we forget about this and add another way to set the HWP direction for a cal block.